pkgs/servers/nextcloud/default.nix

```diff
@@ -1,17 +1,14 @@
 { lib, stdenv, fetchurl, nixosTests
 , nextcloud27Packages
 , nextcloud26Packages
-, nextcloud25Packages
 }:
 
 let
   generic = {
     version, hash
   , eol ? false, extraVulnerabilities ? []
   , packages
-  }: let
-    major = lib.versions.major version;
-  in stdenv.mkDerivation rec {
+  }: stdenv.mkDerivation rec {
```

Comment:
```suggestion
  }: stdenvNoCC.mkDerivation rec {
```

I have this locally for a longer time and thought I suggest it while I fix my merge conflict

Reply:
Can you please explain what kind of benefits should we see thanks to using this stdenv?

Reply:
I'm not sure, but I think the advantage is this avoid rebuilding on compiler changes (please correct me if wrong)

Reply:
Thank you, I know what is `stdenvNoCC`, but that does not help me to understand the benefits, if a compiler changes, the whole tree will change anyway.

Reply:
It is not really important but since we are not using any compilers we might as well not download them or check for makefiles or similar.

Reply:
Fixed :shrug: